cmd/common-main.go

```diff
@@ -260,6 +260,10 @@ func initConsoleServer() (*restapi.Server, error) {
 		return nil, err
 	}
 
+	subPath := restapi.GetSubPath()
```

Comment:
Is this safe to change base paths in this manner?

Reply:
The spec is automatically generated by the `swagger API`. Therefore, we can only inject the desired base path when the minio server is starting. The `GetSubPath ` function has a validity check when reading environment, so I think it's safe.